packages/core/src/App/Containers/EffortlessLoginModal/effortless-login-modal.tsx

```diff
@@ -16,12 +16,31 @@ const EffortlessLoginModal = observer(() => {
     const portal_element = document.getElementById('effortless_modal_root');
     const history = useHistory();
     const { client } = useStore();
-    const { setShouldShowEffortlessLoginModal } = client;
+    const { setShouldShowEffortlessLoginModal, passkeysTrackActionEvent } = client;
+
+    React.useEffect(() => {
+        if (!portal_element) return;
+        passkeysTrackActionEvent({ action: 'open' }, true);
+
+        const track_close = () => {
```

Comment:
do we need this function declaration inside useeefect?

Reply:
why not?
it is used only here, and we can remove eventlistener here.